source/jsonnet/england-wales/ccs/blocks/who-lives-here/anyone_else_list_collector.jsonnet

```diff
@@ -182,37 +181,65 @@ local editQuestion(questionTitle) = {
   add_block: {
     id: 'add-person',
     type: 'ListAddQuestion',
-    question: {
-      id: 'add-question',
-      type: 'General',
-      title: {
-        text: 'Who do you need to add?',
-        placeholders: [
-          placeholders.address,
-        ],
-      },
-      instruction: 'Enter a full stop (.) if the respondent does not know a person’s “First name” or “Last name”',
-      answers: [
-        {
-          id: 'first-name',
-          label: 'First name',
-          mandatory: true,
-          type: 'TextField',
-        },
-        {
-          id: 'middle-names',
-          label: 'Middle names',
-          mandatory: false,
-          type: 'TextField',
+    question_variants: [
+      {
+        question: {
+          id: 'add-question',
+          type: 'General',
+          title: 'What is your full name?',
+          answers: [
+            {
+              id: 'first-name',
+              label: 'First name',
+              mandatory: true,
+              type: 'TextField',
+            },
+            {
+              id: 'middle-names',
+              label: 'Middle names',
+              mandatory: false,
+              type: 'TextField',
+            },
+            {
+              id: 'last-name',
+              label: 'Last name',
+              mandatory: true,
+              type: 'TextField',
+            },
+          ],
         },
```

Comment:
We should make this a partial and pass in the title, the rest is identical. Same way the edit block is done.

https://github.com/ONSdigital/eq-questionnaire-schemas/blob/wlh-variant-and-driver-question/source/jsonnet/england-wales/ccs/blocks/who-lives-here/anyone_else_list_collector.jsonnet#L244-L257

Reply:
The only difference is lack of `instruction` block in one of the variants.

Reply:
Ya, so we should be able to deduplicate the rest and add an instruction key to the single variant.